vault-kong.sh

```diff
@@ -69,7 +69,7 @@ houseKeeping # temp files and payloads
 # Generate Kong PKI/TLS materials if they haven't been already...
 if [[ (! -f ${_KONG_PEM}) || (! -f ${_KONG_SK}) ]]; then
     echo ">> (3) Create PKI materials for Kong TLS server certificate"
-    /vault/pki-setup.sh /vault/pki-setup-config-kong.env
+    ${_VAULT_DIR}/pki-setup.sh ${_VAULT_DIR}/pki-setup-config-kong.env
```

Comment:
Now that I think about it, considering someone might eventually want to change the `pki-setup-config-kong.env` file, can we have that as a separate environment variable? Then that file can live inside `$SNAP_DATA` for the snap and be modifiable by users.